style.css

```diff
@@ -413,7 +409,108 @@ body {
   font-weight: normal;
 }
 
-.btn-link {
-  border: 1px solid red;
-  float: left;
+/* contact form */
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  width: 100%;
+  height: 750px;
+  border: none;
+  left: 0;
+  top: 1rem;
+  border-top-left-radius: 60px;
+}
+
+.form-header {
+  font-size: 40px;
+  font-weight: 800;
+  line-height: 52px;
+  color: #fff;
+  text-align: center;
+  padding-top: 70px;
+}
+
+.form-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  margin-left: 30px;
+  margin-top: 2rem;
+  max-width: 400px;
+}
+
+li {
+  list-style-type: none;
+}
+
+input {
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  height: 48px;
+  width: 327px;
+  margin-top: 1rem;
+  display: flex;
+  outline: none;
+  gap: 1.5rem;
+  justify-content: center;
+  margin-left: 30px;
+  border-radius: 8px;
+}
+
+input::placeholder {
+  padding: 12px;
+  color: #172b4d;
+}
+
+textarea {
+  width: 350px;
+  height: 180px;
+  margin-top: 2rem;
+  margin-left: 30px;
+  border-radius: 8px;
+  border: none;
+}
```

Comment:
[OPTIONAL] this is only optional because It was not mentioned by the previous reviewer. However, I strongly recommend that to apply the changes in order for you not to have any issues when you start doing the desktop version.

The contact form is not responsive, the `textarea` overlaps outside of the form background. There's also a huge gap on the right in other mobile devices. Kindly look into this.
![image](https://user-images.githubusercontent.com/38088818/165464691-70d64b7d-3ed0-491e-8921-3e711284375d.png)